Load individual elements if state dict load fails (#5213)
Unity-Technologies/ml-agents: docs/Training-ML-Agents.md

want to run inference in Unity, you should use the
[Unity Inference Engine](Getting-Started.md#running-a-pre-trained-model).

Additionally, if the network architecture changes, you may still load an existing model,

> and ML-Agents will only load the parts of the model that haven't changed. For instance,

Reviewer: This is ambiguous because the user does not know which parts would change. Adding a new reward signal requires the critic to be re-initialized for example.

Reviewer: ```suggestion
but ML-Agents will only load the parts of the model it can load and ignore all others. For instance,
```
This is still ambiguous but the user will not try to interpret what "it can load" means (while they will try to interpret "the parts that haven't changed")
